README.rst

```diff
@@ -33,140 +106,69 @@ bpython-urwid
 
 * urwid
 
-Introduction
-============
-A few people asked for stuff like syntax highlighting and autocomplete for the
-Python interactive interpreter.  IPython seems to offer this (plus you can get
-readline behaviour in the vanilla interpreter) but I tried IPython a couple of
-times. Perhaps I didn't really get it, but I get the feeling that the ideas
-behind IPython are pretty different to bpython. I didn't want to create a whole
-development environment; I simply wanted to provide a couple of neat features
-that already exist and turn them into something a little more interactive.
-
-The idea is to provide the user with all the features in-line, much like modern
-IDEs, but in a simple, lightweight package that can be run in a terminal
-window, so curses seemed like the best choice.  Sorry if you use Windows.
-
-bpython doesn't attempt to create anything new or groundbreaking, it simply
-brings together a few neat ideas and focuses on practicality and usefulness.
-For this reason, the "Rewind" function should be taken with a pinch of salt,
-but personally I have found it to be very useful. I use bpython now whenever I
-would normally use the vanilla interpreter, e.g. for testing out solutions to
-people's problems on IRC, quickly testing a method of doing something without
-creating a temporary file, etc..
-
-I hope you find it useful and please feel free to submit any bugs/patches (yeah
-right)/suggestions to:
-robertanthonyfarrell@gmail.com
-or place them at the github issue page for this project at:
-http://github.com/bpython/bpython/issues/
+==========
+Known Bugs
+==========
+For known bugs please see bpython's `issues tracker`_ at github.
 
-For any other ways of communicating with bpython users and devs you can find us
-at the community page on the projects homepage:
-http://bpython-interpreter.org/community
+======================
+Contact & Contributing
+======================
+I hope you find it useful and please feel free to submit any bugs/patches
+suggestions to `Robert Athony Farrell`_ or place them on the github
```

Comment:
@bobf: Do you still want to receive bugs/patches per mail or should we just refer to the bug tracker and the mailing list?